seccomp/tracer.go

```diff
@@ -372,7 +372,7 @@ func (t *syscallTracer) connHandler(c *net.UnixConn) error {
 		// Return here to exit this goroutine in case of error as that implies
 		// that the seccomp-fd is not valid anymore.
 		if err := t.seccompSessionRead(seccompSession); err != nil {
-			logrus.Warnf("Failed to wait for seccomp session: %v", err)
+			logrus.Debugf("Failed to wait for seccomp session: %v", err)
```

Comment:
Why this change ... isn't this a serious condition?

Reply:
Not really. This simply indicates that the seccomp-fd isn't available (at least for polling purposes) anymore, which happens coz the sys-container died, or the seccomp-fd simply ceased to exist after user exited a `docker exec` session.